api/src/main/java/io/strimzi/api/kafka/model/Kafka.java

```diff
@@ -129,6 +133,15 @@ public void setSpec(KafkaSpec spec) {
         this.spec = spec;
     }
 
+    @Description("The status of the Kafka and Zookeeper clusters, and Topic Operator.")
```

Comment:
Isn't `status.conditions` the usual place for the conditions? It might be more common with others.

Reply:
Yes, I prefer the latter suggestion. I just hadn't really considered the need for the CO to publish any kind of status for the TO before now. If we can publish everything we need as conditions (and I think we can) that's fine.

Reply:
Honestly, I can imagine it being just one status for the whole resource - not separate ones. But I guess I can live with both. But for example today we do not track EO status, so that would be for the future.

Reply:
I prefer the latter as well, keeping a single list of conditions per resource